src/com/aca/Manager.java

```diff
@@ -0,0 +1,142 @@
+package com.aca;
+
+import com.aca.types.AudioRecorder;
+import com.aca.types.Gadget;
+import com.aca.types.MemoryGadget;
+import com.aca.types.VideoRecorder;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+import java.util.logging.Logger;
+
+public class Manager {
+    private final List<Gadget> gadgets;
+    private int soldAmount;
+    private List<Gadget> workingGadgets=new ArrayList<>();
+    private List<Gadget> notWorkingGadgets=new ArrayList<>();
+    private Logger logger= Logger.getGlobal();
+
+
+    public Manager(List<Gadget> gadgets) throws FileNotFoundException {
+        this.gadgets = gadgets;
+    }
+
+    public void sellItem(String name) throws IOException {
+        for (Gadget gadget : gadgets) {
+            if (gadget.getName().equals(name)){
+                gadget.setCount(gadget.getCount()-1);
+                soldAmount += gadget.getPrice();
+                Gadget.allItemsCount-=1;
+                int memory = 0;
+                if (gadget instanceof MemoryGadget){
+                    memory = ((MemoryGadget) gadget).getMemoryInGB();
+                }
+                String[] row = new String[]{
+                        gadget.getName(),
+                        String.valueOf(gadget.getDuration()),
+                        String.valueOf(gadget.getPrice()),
+                        String.valueOf(gadget.getCount()),
+                        String.valueOf(memory)
+                };
```

Comment:
put details as string implementing toString() in the models, if we sell red pen audio recorder, I want to have details about it also